content-src/asrouter/templates/OnboardingMessage/_OnboardingMessage.scss

```diff
@@ -139,7 +139,7 @@
   display: inline-block;
   vertical-align: middle;
 
-  @media(max-width: 850px) {
+  @media(max-width: 865px) {
```

Comment:
You might want to add a comment about how this accounts for cards wrapping to the next line